exercises/collatz-conjecture/README.md

```diff
@@ -13,34 +13,32 @@ Given a number n, return the number of steps required to reach 1.
 
 Starting with n = 12, the steps would be as follows:
 
-0. 12
-1. 6
-2. 3
-3. 10
-4. 5
-5. 16
-6. 8
-7. 4
-8. 2
-9. 1
+0.  12
```

Comment:
```suggestion
1.  12
```

Shouldn't this be more consistent? Will your Markdown formatter insist on starting at `0.` and then have multiple `1.`s?